src/components/MoviesPage/__snapshots__/MoviesPage.test.js.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`MoviesPage should render MoviesPage 1`] = `
+<section
+  className="moviePage"
+>
+  <section
+    className="userProfile"
+  >
+    <UserProfile
+      0="T"
+      1="h"
+      2="e"
+      3=" "
+      4="R"
+      5="o"
+      6="c"
+      7="k"
```

Comment:
I guess there is some error with rest operator when you  add props to UserProfile component

Reply:
Cool, fixed it with user={{name: "The Rock"}} as per your suggestion